scripts/publish/src/getRegistry.js

```diff
@@ -0,0 +1,7 @@
+const { exec } = require('shelljs');
+
+// Get version of package in NPM regsitry. Returns 0.0.0 if doesn't exist.
+module.exports = name => {
+  const npmVersion = exec(`npm show ${name} version`, { silent: true }).stdout;
```

Comment:
Would be good to use async/await whenever you're executing commands and need the response:

```
module.exports = async name => {
    const npmVersion = await exec(`npm show ${name} version`, { silent: true })
...
```